tests/integration/launch_container.sh

```diff
@@ -213,6 +213,9 @@ elif $is_sm_neo_context; then
     -e SM_NEO_CACHE_DIR=/opt/ml/compilation/cache \
     -e SM_NEO_HF_CACHE_DIR=/opt/ml/compilation/cache \
     -e COMPILER_OPTIONS={} \
+    -e AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID \
+    -e AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY \
+    -e AWS_SESSION_TOKEN=$AWS_SESSION_TOKEN \
```

Comment:
@siddvenk  Why do we need explicit setting? -v ~/.aws:/root/.aws \ this is supposed to work, I believe. Do we need session token now

Reply:
This is just to make it easier for local testing

Reply:
Oh got it. Thanks!